code/modules/mob/living/carbon/human/human.dm

```diff
@@ -943,6 +943,10 @@
 
 	..()
 
+/mob/living/carbon/human/proc/is_heart_broken()
```

Comment:
```suggestion
/// Returns whether this person has a broken heart but is otherwise revivable
/mob/living/carbon/human/proc/is_heart_broken()
```

We also ought to test `is_revivable` too but ignore heart status from it because they could be decapped too.